README.md

```diff
@@ -136,6 +136,58 @@ export default {
 </script>
 ```
 
+### File Upload Sample
+You can upload file instead of using direct image link.
+*Usage*
+In your form add a `file input` along with `vue-croppie` component.
+```html
+<input :class="form-control"  type="file" @change="croppie($event)/>
```

Comment:
Why is the class using data binding? `:class="form-control"` The class is unnecessary for an example, especially that this class is from bootstrap I presume?

Reply:
In the `@change` event. Please do just `@change="croppie"` the event object is being passed implicitly in the method so you can just get the event in the method declaration.

Reply:
The `@change="croppie($event)` is missing a closing quote.